src/Gui/ViewProviderOrigin.h

```diff
@@ -76,6 +76,12 @@ class GuiExport ViewProviderOrigin : public ViewProviderDocumentObject
 
     /// Returns default size. Use this if it is not possible to determine appropriate size by other means
     static double defaultSize();
+
+    // the factor by which the axes are longer than the planes
+    static constexpr float axesScaling = 1.5f;
+
+    // default color for origin (light-blue stored as 0xRRGGBBAA)
+    static const uint32_t defaultColor = (50<<24) + (150<<16) + (250<<8) + 255;
```

Comment:
maybe this works:
```suggestion
    static const uint32_t defaultColor = 0x3296faff;
```
I think storing as hex is more easily understood as this is standard format for colors

Reply:
I think mere mortal humans are more familiar with the decimal system (RGB for light-blue = 50, 150, 250) but I am not obsessed with this. If you like to store it as a hex value, I don't mind.

Reply:
the syntax with sums and << operators obscures this to regular people IMO, hex representations of color are very common on any graphic editing software